### PR TITLE
Replace oscoin-ledger with radicle-registry

### PR DESCRIPTION
**Pull request by geigerzaehler**
Replace oscion-ledger with radicle-registry

### CODE REVIEW

File: proxy/src/main.rs [hunk 1]
```diff
@@ -27,15 +27,15 @@ mod server_warp;
 mod source;
 
 fn main() {
-    std::env::set_var("RUST_BACKTRACE", "full");
```
Comment (xla) [inline proxy/src/main.rs]: I'd like to keep that, esp. during our rapid development it's helpful, we can consider disabling it for release builds.

File: proxy/src/schema.rs [hunk 2]
```diff
@@ -95,16 +95,16 @@ juniper::graphql_scalar!(AccountId where Scalar = <S> {
     description: "AccountId"
 
     resolve(&self) -> Value {
-        Value::scalar(hex::encode(self.0))
+        Value::scalar(hex::encode(self.0.as_ref() as &[u8]))
     }
 
     from_input_value(v: &InputValue) -> Option<AccountId> {
-        let mut bytes = [0_u8; 20];
+        let mut bytes = [0_u8; 32];
```
Comment (xla) [inline proxy/src/schema.rs]: Such bytes!

File: proxy/src/main.rs [hunk 3]
```diff
@@ -27,15 +27,15 @@ mod server_warp;
...
-    std::env::set_var("RUST_LOG", "info");
+    // std::env::set_var("RUST_BACKTRACE", "full");
+    // std::env::set_var("RUST_LOG", "info");
     pretty_env_logger::init();
 
     let source_type = std::env::args().nth(1).expect("no source was given");
 
     let context = match source_type.as_ref() {
         "ledger" => {
```
Comment (xla) [inline proxy/src/main.rs]: Probably makes sense to call this flag `registry` now.

File: proxy/src/main.rs (hunk 1 above)
Comment (geigerzaehler) [reply proxy/src/main.rs]: I did not intend to commit this. Will revert.

**Comment by geigerzaehler** [comment]
Can you have another look @xla. I adressed all comments and updated the revision to master.

**Comment by xla** [comment]
@geigerzaehler Looks extra dope. ready to be merged. :)

**Pull request merged**